addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2735,6 +2735,51 @@ def test_out_invoice_change_period_accrual_1(self):
             {'amount_currency': 120.0,  'debit': 60.0,  'credit': 0.0,      'account_id': wizard.revenue_accrual_account.id,        'reconciled': True},
         ])
 
+    @freeze_time('2019-01-16')
+    def test_out_invoice_change_period_past_move_date(self):
+        move = self.env['account.move'].create({
+            'move_type': 'out_invoice',
+            'journal_id': self.company_data['default_journal_sale'].id,
+            'invoice_date': '2019-01-01',
+            'partner_id': self.partner_a.id,
+            'invoice_line_ids': [
+                Command.create({
+                    'account_id': self.company_data['default_account_revenue'].id,
+                    'price_unit': 1000,
+                }),
+            ]
+        })
+        move.action_post()
+
+        context = {
+            'default_move_type': 'out_invoice',
+            'active_model': 'account.move.line',
+            'active_ids': move.mapped('invoice_line_ids').ids
+        }
+        wizard = self.env['account.automatic.entry.wizard'] \
+            .with_context(context) \
+            .create({
+                'action': 'change_period',
+                'journal_id': self.company_data['default_journal_misc'],
+                'revenue_accrual_account': self.env['account.account'].create({
+                    'name': 'Accrual Revenue Account',
+                    'code': '765432',
+                    'account_type': 'expense',
+                    'reconcile': True,
```

Comment:
could you use an existing account?
Or just duplicate the default_revenue account
(Accrual Revenue Account should not be of type `expense` right? and does it need to be reconcilable?